cli/config_cli.h

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+#ifndef CLN_TEST
+// Redefinition procedure it is a very cool feature
+// However, if we try to redefine a feature that is already
+// defined like read in alpine we can have triky compilation error
```

Comment:
Nit: We use `/* ... */` comments everywhere.

Reply:
Ops, we are not in rust :)